src/managers/conda/condaUtils.ts

```diff
@@ -842,3 +1012,50 @@ export async function getCommonCondaPackagesToInstall(
     const selected = await selectCommonPackagesOrSkip(common, installed ?? [], !!options.showSkipOption);
     return selected;
 }
+
+async function installPython(
```

Comment:
is it possible to give a version to install? Or is this like quick install?

Reply:
This is like quick install